packages/ui/src/services/i18n/dict/en/bounty.json

```diff
@@ -377,4 +378,4 @@
     "title": "You are slashed",
     "description": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis"
```

Comment:
It's good to also replace this text with a meaningful one. @dmtrjsg wdyt?

Reply:
Yes but it's done in #3080